Makefile

```diff
@@ -146,7 +146,7 @@ update/docsy:
 
 .PHONY: gen/code
 gen/code:
-	docker run --rm -v ${PWD}:/repo -it gcr.io/pipecd/codegen:0.8.0 /repo
+	docker run --rm -v ${PWD}:/repo -it ghcr.io/pipe-cd/codegen:v0.32.0 /repo
```

Comment:
should we parse the value from `release/RELEASE` file and use it here?

Reply:
from "hack" script
```shell
╰─ cut -d ' ' -f 2 release/RELEASE
v0.32.0
```

Reply:
What a nice idea! Let me apply it. 💯